components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -1,278 +1,289 @@
-import { Injectable } from '@angular/core';
+import { Injectable, OnInit, OnDestroy } from '@angular/core';
 import { Store } from '@ngrx/store';
-import { Observable } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
 
 import { NgrxStateAtom } from 'app/ngrx.reducers';
 import { isIAMv2 } from 'app/entities/policies/policy.selectors';
 import { FeatureFlagsService } from 'app/services/feature-flags/feature-flags.service';
 import { clientRunsWorkflowEnabled } from 'app/entities/client-runs/client-runs.selectors';
 import * as fromClientRuns from 'app/entities/client-runs/client-runs.reducer';
 
-import { MenuItemGroup } from './layout.model';
+import { Sidebars } from './layout.model';
+
+import {
+    UpdateSidebars
+} from './layout.actions';
 
 @Injectable({
     providedIn: 'root'
 })
-export class LayoutSidebarService {
+export class LayoutSidebarService implements OnInit, OnDestroy {
+    public sidebars = new BehaviorSubject<Sidebars>({});
     public applicationsFeatureFlagOn: boolean;
-    public isIAMv2$: Observable<boolean>;
+    public isIAMv2: boolean;
     public ServiceNowfeatureFlagOn: boolean;
+    private activeSidebar: string;
     private workflowEnabled: boolean;
 
-
     constructor(
         private store: Store<NgrxStateAtom>,
         private clientRunsStore: Store<fromClientRuns.ClientRunsEntityState>,
         private featureFlagsService: FeatureFlagsService
     ) {
-        this.isIAMv2$ = this.store.select(isIAMv2);
         this.applicationsFeatureFlagOn = this.featureFlagsService.getFeatureStatus('applications');
         this.ServiceNowfeatureFlagOn = this.featureFlagsService.getFeatureStatus('servicenow_cmdb');
+        this.store.select(isIAMv2).subscribe(
```

Comment:
Need to add the standard `takeUntil(this.isDestroyed)` before a `subscribe`
both here and on L41 below.